notebooks/index.ipynb

```diff
@@ -0,0 +1,95 @@
+    "\n",
+    "Another great resource to learn Emukit are the [examples](https://github.com/amzn/emukit/tree/develop/emukit/examples) which are more elaborated modules focused either on the implementation of a new method with Emukit components or on the analysis and solution of some specific problem.\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Available Tutorials"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **Getting Started:**\n",
+    "    * [Philosophy and Basic use of the library](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-intro.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **Scientific tutorials:**\n",
+    "    * [Introduction to Bayesian optimization](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-Bayesian-optimization-introduction.ipynb)\n",
+    "    * [Introduction to multi-fidelity Gaussian processes](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-multifidelity.ipynb)\n",
+    "    * [Introduction to sensitivity analysis](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-sensitivity-montecarlo.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **Features tutorials:**\n",
+    "    * [Bayesian optimization with external evaluation of the objective](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-Bayesian-optimization-external-objective-evaluation.ipynb)\n",
+    "    * [Benchmark automatically several Bayesian optimization methods](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-Bayesian-optimization-benchmark.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* **Do it yourself:**\n",
+    "    * [How to write a tutorial model for Emukit](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-how-to-write-a-notebook.ipynb) (please follow the contribution guide) \n",
+    "    * [How to use custom model](http://nbviewer.jupyter.org/github/amzn/emukit/blob/develop/notebooks/Emukit-tutorial-custom-model.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Contribution guide\n",
+    "\n",
+    "We really appreciate contributions. Tutorials and examples are a great way to spread what you have learned about, Emukit across the community and an excellent way to showcase new features. If you want to contribute with a new tutorial please follow [these](https://github.com/amzn/emukit/tree/develop/emukit/examples) steps.\n",
```

Comment:
"Emukit across the community" - is emukit a verb now?

Reply:
changed